### PR TITLE
Niloofar / Fix typescript errors in deriv.com [parent-pr]

### PR DESCRIPTION
**Pull request by niloo-fs**
Changes:

-   TypeScript Errors (668 errors in 226 files)

## Type of change

-   [x] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by NikitK-deriv — COMMENTED** [review]
What a big PR 😮 Great work 👍

File: src/pages/home/_what-our-clients-say.tsx [hunk 1]
```diff
@@ -302,7 +310,7 @@ const WhatOurClientsSay = () => {
                             width="240px"
                             height="100px"
                             tabletL={{
-                                heigth: '64px',
+                                height: '64px',
```
[the remark this answers is not in the document]
Comment (niloo-fs) [reply src/pages/home/_what-our-clients-say.tsx:313]: > What a big PR open_mouth Great work +1

Thank u @NikitK-deriv :smiley: BTW I didn't do all of them myself

File: src/components/elements/carousel/carousel-style.ts [hunk 2]
```diff
@@ -151,14 +160,10 @@ export const StyledDot = styled.div`
     background-color: ${({ color }) => (color ? `var(--color-${color})` : 'var(--color-grey-21)')};
 `
 
-type ChevronRightAndLeftProps = {
-    black?: string | boolean
-    red?: string | boolean
-}
-export const ChevronRight = styled(StyledChevron)<ChevronRightAndLeftProps>`
```
[the remark this answers is not in the document]
Comment (niloo-fs) [reply src/components/elements/carousel/carousel-style.ts:158]: As I checked we don't use these properties in the styles we defined so I changed that with color property, and I changed them in the academy too and asked the QA team to test that. if it didn't affect anything so we can remove it :smiley:

[the remark this answers is not in the document]
Comment (niloo-fs) [reply src/components/elements/carousel/carousel-style.ts:158]: If I need to add the comment will do that

File: src/pages/home/_what-our-clients-say.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (niloo-fs) [reply src/pages/home/_what-our-clients-say.tsx:313]: Yes you are right, BTW I asked QA to test it, if we don't need that I will remove it later :blush:

**Comment by niloo-fs** [comment]
> What a big PR open_mouth Great work +1

Thanks @NikitK-deriv :relaxed: BTW I didn't do all of them myself, I had some helps too 
Also thanks for reviewing this and your comments :pray:

File: src/components/elements/carousel/carousel-style.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (NikitK-deriv) [reply src/components/elements/carousel/carousel-style.ts:158]: Clear, thanks 🙇

File: src/pages/home/_what-our-clients-say.tsx [hunk 3]
```diff
@@ -259,7 +267,7 @@ const ClientSlide = ({ quote, name }: ClientSideProps) => (
             {quote}
         </ClientTestimonial>
         <Flex direction="column" height="fit-content">
-            <ClientName size={'16px'} weight={700}>
+            <ClientName size={'16px'} weight="700">
```
[the remark this answers is not in the document]
Comment (NikitK-deriv) [reply src/pages/home/_what-our-clients-say.tsx:270]: Cool 👍 good to know

**Review by mohammad-rework — APPROVED** [review]

**Pull request merged**